tic_tac_toe/game.py

```diff
@@ -63,42 +108,137 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    winning_combos = [
+        [(0,0),(0,1),(0,2)],
+        [(1,0),(1,1),(1,2)],
+        [(2,0),(2,1),(2,2)],
+        [(0,0),(1,0),(2,0)],
+        [(0,1),(1,1),(2,1)],
+        [(0,2),(1,2),(2,2)],
+        [(0,0),(1,1),(2,2)],
+        [(0,2),(1,1),(2,0)]
+        ]        
+        
+    for combo in winning_combos:
+        if _is_winning_combination(board, combo, player) == True:
+            return player
+    return None
+            
+    
+    """
+    combination = []
+    num = 1
+    for i in range(3):
+        for j in range(3):
+            #print num
+            if board[i][j] == player:
+                combination.append(num)
+            num += 1
+        
+    return combination
+    """
+
+   
 
 # public interface
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
-
+    game = {
+            'player1': player1,
+            'player2': player2,
+            'board': [
+                ["-", "-", "-"],
+                ["-", "-", "-"],
+                ["-", "-", "-"],
+            ],
+            'next_turn': player1,
+            'winner': None
+        }
+    return game
 
 def get_winner(game):
     """
     Returns the winner player if any, or None otherwise.
     """
-    pass
-
+    return game['winner']
+    
 
 def move(game, player, position):
     """
     Performs a player movement in the game. Must ensure all the pre requisites
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    next_player = get_next_turn(game)
+    if _position_is_valid(position) != True:
```

Comment:
You might consider using "if not this_variable_is_true():" instead of using "!=". I'm pretty sure it's more  pythonic.